spec/features/user_can_checkout_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+
+RSpec.feature "User can checkout tasks" do
+  scenario "they can checkout tasks" do
+    volunteer = create(:volunteer)
+    create(:city_with_tasks)
+
+    visit tasks_path
+
+    expect(page).to have_content("Cart: 0")
+    expect(page).to have_content("LOGIN")
+
+    page.all(".card-action")[0].click_link("Add to Cart")
+    page.all(".card-action")[1].click_link("Add to Cart")
+
+    click_on("Cart: 2")
+
+    expect(page).to have_current_path("/cart")
+    expect(page).to_not contain_exactly("Checkout")
+    click_button("Login or Create Account to Checkout")
+
+    login_volunteer(volunteer)
+
+    expect(page).to have_current_path("/cart")
+    expect(page).to have_content("LOGOUT")
+
+    expect(page).to have_content("Task 1")
+    expect(page).to have_content("Task 2")
+
+    click_on("Checkout")
+
+    expect(page).to have_current_path("/commitments")
+
+    within(".flash-notice") do
+      expect(page).to have_content("Order was successfully placed")
```

Comment:
Should rephrase flash-notice to "Pledge" or "Commitment" rather than using "Order"